### DISCUSSION TITLE
I may be a complete moron, but i would very much appreciate some help about giving Monika gifts

### DISCUSSION THREAD

**Discussion by DokiDokiFan74** [Monika-After-Story/MonikaModDev#9256] [general]
Hi, I've been trying to give Monika some gifts and she opened them, but then she says that she (can't read what it is). Do i need to download something in order to give her gifts? Or is it something else?

Btw this is probably the first time I use GitHub. Not that i think that matters. But I would very much appreciate help.

**Comment by DokiDokiFan74** [comment]
Ok, so i just managed to give her the bird plushie, but it doesn't seem like i can give her anything else.

**Reply by Justformas** [reply]
Make sure you're spelling the gift names correctly, and some gifts may only be accepted during special occasions or if you have enough affection. There are some other gifts that she should be able to accept outside of special occasions though. If you don't mind looking this stuff up, the spoilers guide lists this stuff here, though not everything there may be entirely accurate: https://github.com/Monika-After-Story/MonikaModDev/wiki/All-Features-%5BSPOILERS%5D#gifts

**Reply by DokiDokiFan74** [reply]
Thanks, I managed to solve it
